ui/perfherder/perf-helpers/constants.js

```diff
@@ -39,6 +39,7 @@ export const filterText = {
   hideUncertain: 'Hide uncertain results',
   showNoise: 'Show only noise',
   hideUncomparable: 'Hide uncomparable results',
+  useReplicates: 'Use replicates (try only)',
```

Comment:
@gmierz If you plan on moving the `Use replicates (try only)` next to the JSON I suggest looking into making this option available only for the try comparisons, or perhaps disabling the button for the other repos.